### DISCUSSION TITLE
arabic support

### DISCUSSION THREAD

**Discussion by ostende** [OpenCockpit/TMDBCockpit#2] [General]
hi can you added ar-SA in the configinit please so that can get the result in arabic for arab users

**Comment by xcentaurix** [comment]
sure, will do.

**Comment by xcentaurix** [comment]
the latest version has now "ar" as tmdb lang included.
i can see some arabic characters but i'm not sure whether it works.

**Reply by ostende** [reply]
Thank u if you added the ar into the configinit it will working fine i was added it and it work fine and then i asked you
